backend/src/services/user.service.ts

```diff
@@ -25,16 +25,26 @@ export const getUserByAuthId = async (uid) => {
 };
 
 
-export const deleteUserEncounter = async (encounterId : string) => {
-  const result = await User.updateMany({}, { $pullAll: {encounters: [{ _id: encounterId}]} });
+export const deleteUserPerson = async (personID: String) => {
+  const result = await User.updateMany({ }, { $pullAll: {persons: [{ _id: personID}]}});
 
-  // Check if User has been updated
   if (result.modifiedCount == 1) {
     return true;
   } else {
     return false;
   }
-};
+}
+
+export const deleteUserEncounter = async (encounterID: String) => {
+  const result = await User.updateMany({ }, { $pullAll: {encounters: [{ _id: encounterID}]}});
+
+  if (result.modifiedCount > 0) {
```

Comment:
Is the relationship between User and Encounter one-to-many? If so then an encounter would only ever belong to one User, and `result.modifiedCount == 1` would be more precise?

Reply:
Ah yes will change that